.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/nuget/SFML"]
+	path = tools/nuget/SFML
+	url = git@github.com:SFML/SFML.git
```

Comment:
The `.gitmodules` file doesn't show it, but the submodule is already pointing to the `2.5.x` tag in the SFML repository. You can check this by clicking at the SFML folder in [GitHub's interface](https://github.com/Davipb/CSFML/tree/nuget/tools/nuget) or by cloning the repo with `--recurse` and opening the `.git/modules/tools/nuget/SFML/HEAD` file: Both point to [commit 2f11710abc5aa478503a7ff3f9e654bd2078ebab](https://github.com/SFML/SFML/commit/2f11710abc5aa478503a7ff3f9e654bd2078ebab).

Reply:
If we add SFML as a submodule, then shouldn't we put it at the root (or in a "extlibs" directory) rather than in tools/nuget? It would be part of the CSFML project, not only be used for the NuGet package.

Reply:
Well, right now it's only used by the NuGet package, since I didn't touch any of CSFML's files or cmake scripts, so that's why I put it in the `tools/nuget` folder.

I can move it to another folder if you decide that's better, the exact location of SFML doesn't matter to the NuGet build scripts.

Reply:
And would it be more work for you to *not* include SFML as a submodule? Or can we go without it, as @eXpl0it3r suggests? I think this kind of decision should be discussed in a more general context.

Reply:
It shouldn't be much trouble to not have SFML as a submodule, it would just require an extra command before the build scripts to clone SFML into a temporary folder.

The submodule was really my "first instinct" when thinking about a build that requires another git repository, but I understand how adding a submodule can be a big step for a repo. I'll update the build to clone SFML on-demand.